contributing file translating (#350)
azl397985856/leetcode: CONTRIBUTING.en.md

# Contributing

## Translation

Please pick any work without translation by `submit new issue`. English version and Chinese version are distinguished by file name, e.g. Chinese version file name abc.md, the corresponding English version should be abc.en.md. 

Manual translation instead of machine translation, there is no need to translate the technical jargon.

## Contributing to problems

Please follow the template of "problems", what you need to submit are:

- Problem and solution markdown file

> - Add the question title in README.md

Reviewer: ```suggestion
- Add the link of the solution in README.md
```

> - Add the question title in README.en.md (optional)

Reviewer: ```suggestion
- Add the link of the solution in README.en.md (optional)
```

> - draw.io file or other pictures (optional)

Reviewer: ```suggestion
- draw.io file(xml) or pictures (optional)
```

> Online painting tools like https://excalidraw.com/, draw.io, processon or iPad apps

> ## Contributing to daily

Reviewer: ```suggestion
## Contributing to daily problem
```

> - Please follow the template of "daily" and submit the issue with correct tags (can refer to official tags on LeetCode).

Reviewer: ```suggestion
- Please follow the template of "daily problem" and submit the issue with correct tags (can refer to official tags on LeetCode).
```
